README.md

```diff
@@ -1,17 +1,21 @@
 ## sell-my-services
 
-We help people to offer their services to others easily and quickly. SellMyServices is a platform where a seller can publish his
-services. These services then can be searched by seekers in their nearby locations. 
+An open source content search tool. This is the backend repository of https://sellmyservices.in.
```

Comment:
content search tool?

Reply:
it's the general term for this product. This idea can be made specific e.g. nearby restaurant search tool etc.

Reply:
Content search seems too general to me

Reply:
local services search tool?

Reply:
updated. Lets merge this :)